api-server/src/main/java/org/apache/cassandra/diff/api/services/DBService.java

```diff
@@ -80,8 +80,7 @@ public DBService(JobConfiguration config) {
             "   job_id," +
             "   job_start_time," +
             "   buckets," +
-            "   keyspace_name, " +
-            "   table_names, " +
+            "   keyspace_table_names, " +
```

Comment:
suggestion: we have ["qualified name"](https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/cql3/QualifiedName.java) for keyspace prefixed table names in cassandra, maybe make this `qualified_table_name`?